libraries/ae-inventory-processor/src/main/java/org/metaeffekt/core/inventory/processor/patterns/contributors/NextcloudAppInfoContributor.java

```diff
@@ -67,7 +67,7 @@ public List<ComponentPatternData> contribute(File baseDir, String virtualRootPat
 
             componentPatternData.set(ComponentPatternData.Attribute.COMPONENT_NAME, name);
             componentPatternData.set(ComponentPatternData.Attribute.COMPONENT_VERSION, version);
-            componentPatternData.set(ComponentPatternData.Attribute.COMPONENT_PART, id);
+            componentPatternData.set(ComponentPatternData.Attribute.COMPONENT_PART, id + "-" + version);
```

Comment:
i went with it, since we have that pattern in many contributors. but still you are right, we cannot assume to always have a version. maybe we should go for a standardized pattern and fallback to something for "version", if version is empty or null?

Reply:
To be clarified.